### PR TITLE
Add 2.1 release notes and headline features

### PR DESCRIPTION
**Pull request by tbrisker**

### CODE REVIEW

File: _includes/manuals/2.1/1.2_release_notes.md [hunk 1]
```diff
@@ -19,15 +19,400 @@ Subnets can be now associated with IPAM plugins providing IP address management
...
 is available as a [Smart Proxy plugin](https://github.com/grizzthedj/smart_proxy_ipam).
 
-### Release Notes
+#### Rails 6 upgrade
+
+Foreman has been updated to run on the latest version of Ruby on Rails at the time of the release, 6.0.2. Many plugins have already been updated as well, but there may be issues with any that have not seen a new release in a while.
+
+#### Support installation on EL8 distributions
+
+Foreman installation is now possible on EL8 distributions. The installer has been updated to support changes needed for these and yum repositories now contain packages for EL8. Note that installation is only supported and tested on RHEL and CentOS, packages may or may not work on Oracle Linux or Scientific Linux.
```
Comment (tbrisker) [inline _includes/manuals/2.1/1.2_release_notes.md:28]: This probably requires some changes in other parts of the manual, but i'm not sure what exactly needs to be changed.

File: _includes/manuals/2.1/1.2_release_notes.md [hunk 2]
```diff
@@ -19,17 +19,403 @@ Subnets can be now associated with IPAM plugins providing IP address management
...
+
+#### Switch default web server from Passenger to Puma
+
+The default installation of Foreman will now set up Puma behind Apache reverse proxy instead of Passenger with mod_passenger. This will enable us to use Web Sockets in the future as well as additional benefits with regards to certificate handling and unifying development and production setups.
```
Comment (lzap) [inline _includes/manuals/2.1/1.2_release_notes.md:32]: I'd add info about how to change back to passenger. Note a SELinux boolean will need to be turned on when passenger needs to be used:

    semanage boolean passenger_run_foreman on

File: _includes/manuals/2.1/1.2_release_notes.md [hunk 3]
```diff
@@ -19,17 +19,403 @@ Subnets can be now associated with IPAM plugins providing IP address management
...
+* Fix Rubocop: Layout/EmptyLineBetweenDefs ([#29099](https://projects.theforeman.org/issues/29099))
+* Fix Rubocop: Style/TrailingCommaIn\*Literal ([#29086](https://projects.theforeman.org/issues/29086))
+* Fix Rubocop: Style/SafeNavigation ([#29085](https://projects.theforeman.org/issues/29085))
+* Fix Rubocop: Style/UnlessElse ([#29084](https://projects.theforeman.org/issues/29084))
+* Fix Rubocop: Performance/InefficientHashSearch ([#29083](https://projects.theforeman.org/issues/29083))
+* Fix Rubocop: Layout/TrailingWhitespace ([#29082](https://projects.theforeman.org/issues/29082))
+* Fix Rubocop: Lint/UnneededCopEnableDirective ([#29081](https://projects.theforeman.org/issues/29081))
+* Fix Rubocop: Style/UnneededCondition ([#29078](https://projects.theforeman.org/issues/29078))
+* Fix Rubocop: Style/Encoding ([#29077](https://projects.theforeman.org/issues/29077))
+* Fix Rubocop: Gemspec/OrderedDependencies ([#29076](https://projects.theforeman.org/issues/29076))
```
Comment (lzap) [inline _includes/manuals/2.1/1.2_release_notes.md]: Maybe we can squash these into one?

Comment (tbrisker) [reply _includes/manuals/2.1/1.2_release_notes.md]: done

File: _includes/manuals/2.1/1.2_release_notes.md (hunk 2 above)
Comment (tbrisker) [reply _includes/manuals/2.1/1.2_release_notes.md:32]: I'm going to merge this now as is so it's available for the RC1 announcement, please open a follow-up PR with what needs to be done.

File: _includes/manuals/2.1/1.2_release_notes.md (hunk 1 above)
Comment (ekohl) [reply _includes/manuals/2.1/1.2_release_notes.md:28]: We've also discussed the quality level and not everyone was comfortable with production quality. Given we've made changes right up before branching, it hasn't seen a lot of testing. One suggestion was to label it beta quality, maybe even for 2.1.0 GA.

File: _includes/manuals/2.1/1.2_release_notes.md [hunk 4]
```diff
@@ -19,17 +19,356 @@ Subnets can be now associated with IPAM plugins providing IP address management
...
-* Rackspace compute resource which has been deprecated in 2.0 has now been dropped. Any existing Rackspace compute resources will be deleted as part of the upgrade.
+* Rackspace compute resource which has been deprecated in 2.0 has now been dropped. Any existing Rackspace compute resources will be deleted as part of the upgrade. [More info](https://community.theforeman.org/t/dropping-rackspace-compute-resource-from-foreman-core/17690)
 * The RPM packages are not tested on Scientific Linux or Oracle Linux. The Foreman installation on Scientific Linux or Oracle Linux may or may not work.
+* TLSv1.1 is no longer supported for the Smart Proxy. Clients connecting to the proxy should use TLSv1.2 or newer.
+* PuppetDB module is no longer part of the Foreman installer. Users who use PuppetDB should manage it on their own and not rely on the Foreman installer to do so. [More info](https://community.theforeman.org/t/drop-puppetlabs-puppetdb-from-foreman-installer/17787)
```
Comment (ekohl) [inline _includes/manuals/2.1/1.2_release_notes.md:371]: Thanks for this. It was on my todo list but glad you took the time.

File: _includes/manuals/2.1/1.2_release_notes.md (hunk 1 above)
Comment (tbrisker) [reply _includes/manuals/2.1/1.2_release_notes.md:28]: Can you please add that in a follow up PR as well as the needed changes in the installation instructions? I wasn't very involved in the el8 work so i'm not sure what changes are needed.

**Pull request merged**